### PR TITLE
ITK v5.1rc2 updates

### PR DESCRIPTION
**Pull request by hjmjohnson**

### CODE REVIEW

**Comment by dzenanz** [comment]
I guess merge if CI is green?

**Comment by hjmjohnson** [comment]
@dzenanz Can you please merge this?  I don't have access to kitware repos.

File: test/azure-pipelines.yml [hunk 1]
```diff
@@ -77,11 +77,16 @@ jobs:
       curl -L https://raw.githubusercontent.com/InsightSoftwareConsortium/ITK/dashboard/itk_common.cmake -O
     displayName: 'Fetch CTest driver script'
 
+  - bash: |
```
Comment (dzenanz) [inline test/azure-pipelines.yml]: cookiecutter should not be run in CI, but just once and customized template used to update PhaseSymmetry.

**Comment by dzenanz** [comment]
mkdir: cannot create directory ‘/home/vsts/work/1/ITKModuleTemplate/.git’: No such file or directory

File: test/azure-pipelines.yml [hunk 2]
```diff
@@ -145,75 +162,75 @@ jobs:
...
-      set CC=cl.exe
-      set CXX=cl.exe
-      powershell.exe -file .\windows-download-cache-and-build-module-wheels.ps1
-    displayName: 'Build Python packages'
-
-  - task: PublishPipelineArtifact@0
-    inputs:
-      artifactName: 'WindowsWheels'
-      targetPath: './dist'
+# - job: 'PackageLinux'
```
Comment (dzenanz) [inline test/azure-pipelines.yml]: Is packaging disabled on purpose?

Comment (hjmjohnson) [reply test/azure-pipelines.yml]: Yes.  My understanding was @thewtex was working on a different solution. 

I have re-enabled it.

**Comment by thewtex** [comment]
I will update CI in other branches, then rebase this one.

**Comment by hjmjohnson** [comment]
@thewtex Can this be merged now, or at least the code changes?  @mseng10 is working to remove all compiler warnings in all remote modules, but that process is complicated by these existing outstanding PRs.

File: .clang-format [hunk 3]
```diff
@@ -0,0 +1,151 @@
+## This config file is only relevant for clang-format version 8.0.0
```
Comment (thewtex) [inline .clang-format]: Please do not copy the `.clang-format` repository into the repository. We have a GitHub Action that will fetch the latest version from the ITK repository, which avoids duplication and getting out of sync.

File: test/azure-pipelines.yml [hunk 4]
```diff
@@ -1,6 +1,7 @@
+# The file template originates from the instructions: https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/README.md
```
Comment (thewtex) [inline test/azure-pipelines.yml]: Remove changes to the Azure Pipelines configuration, there are syntax errors.

**Comment by thewtex** [comment]
@hjmjohnson a few inline comments, after which we can merge.

**Comment by hjmjohnson** [comment]
Removed .clang-format, and restored origin/master azure-pipeline.yml file (which points to wrong version of ITK, so likely fails).

**Comment by thewtex** [comment]
@hjmjohnson @dzenanz thanks!

**Pull request merged**